### DISCUSSION TITLE
Block Manager Activity Redesign

### DISCUSSION THREAD

**Discussion by thisisshadow** [Sketchware-Pro/Sketchware-Pro#1159] [Show and tell]
• I renewed the block manager activity on the mod settings page and this design I made was very cute.
![IMG_20240606_101554](https://github.com/Sketchware-Pro/Sketchware-Pro/assets/116355867/aa6bbb9c-8435-41c3-b633-495abdd65deb)

**Comment by RoyTheReal** [comment]
WOAH
